ruby/blue_steel/README.md

```diff
@@ -0,0 +1,51 @@
+# BlueSteel
+
+We all recognize that style matters, and particularly that having _a_ consistent
+style matters more than _my_ or _your_ particular preference for a style (See
+Sandi Metz's _Looks Matter_). This repository contains the Apartment List style.
+
+### Changing Style
+
+Thrash is anathema to consistency, so we are making it very difficult to change
+styles. Consider it a Constitution. We all agreed on it hundreds of years ago,
+and though everyone has some gripes with it, it's pretty good and we're leaving
+it alone.
+
+Since we're intentionally making it difficult to change, I'm only going to give
+you one criterion to consider when you want to propose a change:
+
+1. The style is probably not going to change, and that's ok!
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'blue_steel'
```

Comment:
Will this be a public gem that is published to ruby gems? If not, then the gem installation will need to point to the URL.